frontend/src/components/website/WebsiteForm.vue

```diff
@@ -0,0 +1,157 @@
+                        <p class="inline-element">
+                            <span>Tracking ID: </span>
+                            <span>{{ currentWebsite.tracking_number }}</span>
+                        </p>
+                        <VBtn
+                            @click="update"
+                            color="#3C57DE"
+                            class="update-form-button"
+                        >
+                            Update
+                        </VBtn>
+                    </VForm>
+                    <div>
+                        <div>
+                            <h2>WebSite Tracking</h2>
+                            <p>
+                                This is Global Site tag tracking code for you website.
+                                Copy and Paste this code as the
+                                first item into the &lt;HEAD> of every Webpage you want to track
+                            </p>
+                            <TrackWebsite :tracking-number="currentWebsite.tracking_number" />
+                        </div>
+                    </div>
+                </VContainer>
+            </VFlex>
+        </VContent>
+    </VApp>
+</template>
+
+<script>
+    import TrackWebsite from './TrackWebsite.vue';
+    import { mapGetters, mapActions } from 'vuex';
+    import {GET_CURRENT_WEBSITE} from "../../store/modules/website/types/getters";
+    import {UPDATE_WEBSITE} from "../../store/modules/website/types/actions";
+
+    export default {
+        name: 'WebsiteForm',
+        components: {
+            TrackWebsite
+        },
+        data: () => ({
+            showErrorMessage: '',
+            showSuccessMessage: '',
+            nameRules: [
+                v => !!v || 'Website name is required',
+                v => (v && v.length >= 8) || 'Website name must be correct. Name must be at least 8 characters.'
+            ],
+            localWebsite: {
+                isWebsiteName: undefined
+            }
+        }),
+        computed: {
+            ...mapGetters('website', {
+                currentWebsite: GET_CURRENT_WEBSITE
+            }),
+            isWebsiteName: {
+                set(value) {
+                    this.localWebsite.isWebsiteName = value;
+                },
+                get() {
+                    if(this.localWebsite.isWebsiteName === undefined) {
+                        return this.currentWebsite.name;
+                    }
+                    return this.localWebsite.isWebsiteName;
+                },
+            }
+        },
+        methods: {
+            ...mapActions('website', {
+                website: UPDATE_WEBSITE
+            }),
+            update() {
+                this.website({
+                    name:this.localWebsite.isWebsiteName
+                }).then((e) => {
+                    this.showSuccessMessage = 'Name Success Save';
+                    this.localWebsite.isWebsiteName = e.name.name
```

Comment:
not sure that you need this, since this key will contain this data, if you want to force component use data from Vuex store it's probably better just set `undefined` here